src/hotspot/share/gc/shenandoah/shenandoahUtils.hpp

```diff
@@ -64,11 +64,12 @@ class ShenandoahTimingsTracker : public StackObj {
 
   ShenandoahPhaseTimings* const         _timings;
   const ShenandoahPhaseTimings::Phase   _phase;
+  const bool _should_aggregate;
```

Comment:
One really tiny thing: `_should_aggregate` should be indented like other field names.